src/Sdk/DTPipelines/workflow-v1.0.json

```diff
@@ -401,10 +401,23 @@
       ],
       "one-of": [
         "non-empty-string",
-        "container-mapping"
+        "services-container-mapping"
       ]
     },
 
+    "services-container-mapping": {
+      "mapping": {
+        "properties": {
+          "image": "non-empty-string",
+          "options": "non-empty-string",
+          "env": "container-env",
+          "ports": "sequence-of-non-empty-string",
+          "volumes": "sequence-of-non-empty-string",
+          "credentials": "container-registry-credentials"
+        }
+      }
+    },
+
```

Comment:
We don't need the section 'services-container-mapping', could you please remove it? :)